### PR TITLE
Add proteomiqon_labeledproteinquantification

### PR DESCRIPTION
**Pull request by ZimmerA**
The tool LabeledProteinQuantification estimates protein abundances using quantified peptide ions.

### CODE REVIEW

File: tools/proteomiqon_labeledproteinquantification/aggregationparams.xml [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+                    <param name="Value" type="float" value="0.0" optional="true" label="Value" />
+                </when>
+                <when value="MultiplyBy">
+                    <param name="Value" type="float" value="0.0" optional="true" label="Value" />
+                </when>
+            </conditional>                                    
+        </section>
+        <section name="LabeledSingleFilters" title="Labeled Single filters">
+            <section name="Light" title="Light">
+                <param name="BiggerThan" type="float" value="" optional="true" label="Is bigger than" />
```
Comment (bgruening) [inline tools/proteomiqon_labeledproteinquantification/aggregationparams.xml:85]: We should add a widget to Galaxy that supports those kinds of elements :)

File: tools/proteomiqon_labeledproteinquantification/labeledproteinquantification.xml [hunk 2]
```diff
@@ -0,0 +1,2527 @@
...
+                </when>        
+            </conditional>
+        </section>
+        <section name="AggregateToProteinGroupsParams" title="Aggregate To Protein Groups Parameters">
+            <expand macro="AggregationParams" />
+        </section>
+        <param name="outputParamfile" type="boolean" value="false" label="Output parameter file"/>
+    </inputs>
+    <outputs>
+        <data format="txt" name="out_labeledQuant" />
```
Comment (bgruening) [inline tools/proteomiqon_labeledproteinquantification/labeledproteinquantification.xml:1240]: Is that a tabular?

**Review by bgruening — APPROVED** [review]

File: tools/proteomiqon_labeledproteinquantification/aggregationparams.xml (hunk 1 above)
Comment (ZimmerA) [reply tools/proteomiqon_labeledproteinquantification/aggregationparams.xml:85]: Yeah, that would come in handy!

File: tools/proteomiqon_labeledproteinquantification/labeledproteinquantification.xml (hunk 2 above)
Comment (ZimmerA) [reply tools/proteomiqon_labeledproteinquantification/labeledproteinquantification.xml:1240]: Well, the tool produces a .txt but I suppose the content is tabular anyways, so I will change it to tabular!

Comment (bgruening) [reply tools/proteomiqon_labeledproteinquantification/labeledproteinquantification.xml]: If its tab separated its a tabular :)

**Review by bgruening — APPROVED** [review]

**Pull request merged**